powerhome/playbook
# [PLAY-783] Converting Title Detail kit to Typescript

**What does this PR do?**
Converting Title Detail kit to Typescript.

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-783)

**How to test?**
1. Go to Title Detail kit on Playbook docs
2. Ensure all the examples still work

#### Checklist:
